lib/main.dart

```diff
@@ -108,6 +118,7 @@ class _RandomWordsState extends State<RandomWords> {
           }
         });
       },
+      tileColor: tileColor,
```

Comment:
i have some idea, how to improve your feature: [[link](https://stackoverflow.com/questions/51340588/flutter-how-can-i-make-a-random-color-generator-background)]
pls, max, remove your infinity cycle & add this one
'Colors.primaries[Random().nextInt(Colors.primaries.length)]' p.s. this works 💯

Reply:
How do I make colorBrightness then light?